x/wasm/keeper/test_common.go

```diff
@@ -811,3 +811,8 @@ func keyPubAddr() (crypto.PrivKey, sdk.AccAddress) {
 	addr := sdk.AccAddress(pub.Address())
 	return key, addr
 }
+
+// ptr returns a pointer to the value passed in
```

Comment:
To my knowledge this convention is not so used in Go.
Maybe we could remove it, if it is not used in many places. WDYT?

Reply:
We can remove it, but then the test gets much more verbose because I have to create a variable first for every value. Well, I guess it's not as bad for my specific test, since I use the same value for all of the fields in my test, so I'll remove it.

Reply:
yeah, I thought so as well.. but since all the values are the same we can just create one variable.
If needed we can introduce this piece of code in the future.